eox_nelp/pearson_vue/pipeline.py

```diff
@@ -613,3 +614,43 @@ def get_enrollment_from_id(enrollment_id, enrollment=None, **kwargs):  # pylint:
             pass
 
     return {}
+
+
+def save_audit_pipeline(pipeline_backend, last_step, cdd_import=None, ead_import=None, **kwargs):
+    """This is not a pipe step, could be but its no the intention because inside a pipeline could
+    be more than one import. so this is a helper for pipelines.
+
+    Args:
+        pipeline_backend (str):  pipeline class name
+        last_step (str): last function executed
+        cdd_import (dict, optional): Defaults to None.
+        ead_import (dict, optional):  Defaults to None.
```

Comment:
Why ? this implementation is too specific, you are assuming in advanced that the backend will implement cdd and ead pipelines steps and that is not true, probably your answer will include that there are some  default values  but again this code was written to work with something specific instead of a general solution